bool set
projeto_semear/Pose pose
---
projeto_semear/Pose pose
